params/config.go

```diff
@@ -339,9 +339,10 @@ type ChainConfig struct {
 	// Astria Specific Configuration
 	AstriaOverrideGenesisExtraData       bool          `json:"astriaOverrideGenesisExtraData,omitempty"`
 	AstriaExtraDataOverride              hexutil.Bytes `json:"astriaExtraDataOverride,omitempty"`
-	AstriaSequencerInitialHeight         *big.Int      `json:"astriaDequencerInitialHeight,omitempty"`
-	AstriaDataAvailabilityInitialHeight  *big.Int      `json:"astriaDataAvailabilityInitialHeight,omitempty"`
-	AstriaDataAvailabilityHeightVariance *big.Int      `json:"astriaDataAvailabilityHeightVariance,omitempty"`
+	AstriaRollupName                     string        `json:"astriaRollupName,omitempty"`
+	AstriaSequencerInitialHeight         uint32        `json:"astriaDequencerInitialHeight,omitempty"`
+	AstriaDataAvailabilityInitialHeight  uint32        `json:"astriaDataAvailabilityInitialHeight,omitempty"`
+	AstriaDataAvailabilityHeightVariance uint32        `json:"astriaDataAvailabilityHeightVariance,omitempty"`
```

Comment:
```suggestion
	AstriaCelestiaHeightVariance uint32        `json:"astriaCelestiaHeightVariance,omitempty"`
```
here `omitempty` is probably fine, albeit strict.